dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.openapi-3.1.feature

```diff
@@ -1,7 +1,7 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=io.openliberty.org.eclipse.microprofile.openapi-3.1
 singleton=true
--features=io.openliberty.mpCompatible-5.0
+-features=io.openliberty.mpCompatible-5.0; ibm.tolerates:="6.0"
```

Comment:
```suggestion
-features=io.openliberty.mpCompatible-6.0
```
Shouldn't this be only EE10 features.

Reply:
It should be, but it can't be yet because it's beta and the EE10 features are not.